indigo_app/forms.py

```diff
@@ -218,12 +218,7 @@ class Meta:
                                             required=False)
     workflows = forms.ModelMultipleChoiceField(queryset=Workflow.objects, widget=forms.CheckboxSelectMultiple,
                                                required=False)
-
-
-class TaskEditForm(TaskCreateForm):
-    class Meta:
-        model = Task
-        fields = ('title', 'description', 'labels', 'workflows')
+    assigned_to = forms.ModelChoiceField(queryset=User.objects, empty_label='Unassigned', required=False)
```

Comment:
Actually, I don't think this should be here.

Reply:
you're totally right! we moved that to the detail view in the meantime, thanks for catching